static/src/stylesheets/layout/nav/_cta-bar.scss

```diff
@@ -5,7 +5,7 @@
     }
 }
 
-.new-header__cta-bar-aus-moment {
```

Comment:
use of `new-header__cta-bar-aus-moment` has been removed, but conveniently this was also for supporters so we can resuse the style here